kernel/src/tests.rs

```diff
@@ -97,12 +97,14 @@ fn test_timer_interrupt() -> TestResult {
 fn test_pagetable_remap() -> TestResult {
     info!("Testing the remapping capabilities of our pagetable...");
 
-    let page_src = alloc_pages(1).unwrap();
+    let page_src = globals::PHYSICAL_MEMORY_MANAGER.alloc(1).unwrap();
+    let page_src = unsafe { slice::from_raw_parts_mut(page_src as *mut u8, PAGE_SIZE) };
```

Comment:
Maybe we should add a wrapper for that because we do this casting in a lot of different places